doc/inferring_metrics.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Could you add an example where you show how xgcm actually does the multiplication of dx*dy if the area is not available?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I've add one example of inferring `rAz` when it is not provided.  However, surface integral with original `rAz` is not close to the one with new `rAzNew=dxV*dyU`.  While I know that dx*dy is only an approximation on the spherical earth, I am not sure how large the difference could be.

Reply:
I cannot push the changes.  After commit, I tried `git push` and it says:
```
fatal: The current branch addDoc has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin addDoc
```
tried with `git push --set-upstream origin addDoc`, it says:
```
warning: redirecting to https://github.com/miniufo/xgcm/
To http://github.com/miniufo/xgcm
 ! [rejected]        addDoc -> addDoc (non-fast-forward)
error: failed to push some refs to 'http://github.com/miniufo/xgcm'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```
I am confused...

Reply:
<p>This might work itself out when you move the notebook to xgcm-examples? If not ping me and we will work it out!</p>

Reply:
I just move the it to `xgcm-examples`.  Hope everything is fine.